src/dev/i18n/extract_default_translations.js

```diff
@@ -7,9 +7,10 @@
  */
 
 import path from 'path';
+import globby from 'globby';
 
 import { extractCodeMessages } from './extractors';
-import { globAsync, readFileAsync, normalizePath } from './utils';
+import { readFileAsync, normalizePath } from './utils';
```

Comment:
@Bamieh you might want to take a look at these changes and verify they're ok.

Reply:
yes LGTM